src/test/java/org/folio/marc4ld/mapper/Bibframe2Marc776IT.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.marc4ld.mapper;
```

Comment:
why not to move this file into ```field776``` folder?

Reply:
Currently we have s structure like that:
<img width="237" alt="image" src="https://github.com/user-attachments/assets/8b20ef8a-a90f-4920-a055-0dea066b1b0d">
so i decided to place it in the root of org.folio.marc4ld.mapper. will fix it

Reply:
have moved all the Bibframe2MarcXXX to appropriate folders